spec/features/performance/live_pages_report_spec.rb

```diff
@@ -0,0 +1,112 @@
+RSpec.feature 'Live pages report', type: :feature do
+  scenario 'Navigate to a live pages report from the home page' do
+    given_i_am_logged_in
+    when_i_view_the_home_page
+    then_i_can_navigate_to_a_live_pages_report
+  end
+
+  scenario 'Live pages report shows the total live pages each day' do
+    given_i_am_logged_in
+    and_there_is_historical_page_data
+    when_i_navigate_to_the_live_pages_report
+    then_i_can_see_the_total_live_pages_each_day
+  end
+
+  scenario 'Filtering by organisation' do
+    given_i_am_logged_in
+    and_there_is_historical_page_data
+    when_i_navigate_to_the_live_pages_report
+    then_i_can_filter_the_report_by_organisation
+  end
+
+  def given_i_am_logged_in
+    create(:user)
+  end
+
+  def and_there_is_historical_page_data
+    # Let's assume it is New Year's Day
+    Timecop.freeze(Time.local(2018, 1, 1))
```

Comment:
How is the time unfrozen? I would be keen [to enable `safe_mode`](https://github.com/travisjeffery/timecop#timecopsafe_mode).